src/Mod/Draft/draftguitools/gui_edit.py

```diff
@@ -843,6 +843,20 @@ def deformat_objects_after_editing(self, objs):
                 obj_gui_tools.restore_object_style(obj, self.objs_formats[obj.Name])
 
 
+    def get_specific_object_info(self, obj, pos):
+        """Return info of a specific object at a given position.
+        """
+        selobjs = Gui.ActiveDocument.ActiveView.getObjectsInfo((pos[0],pos[1]))
+        if not selobjs:
+            return
```

Comment:
`getObjectsInfo `can return None, even if i'm not sure that could be the case, since we already know the user is over an object at this point. What can be objected is that someone else could want to use that method in the future for other purpouses, so to me better to leave that check... do you agree?

Reply:
Yeah, maybe better be safe than sorry (though one could argue that getObjectsInfo should just always return a list, possibly an empty one if nothing is found, but if that is not currently the case, maybe indeed leave it like this).